testapi.pm

```diff
@@ -1982,6 +1983,20 @@ sub eject_cd {
     query_isotovideo(backend_eject_cd => \%nargs);
 }
 
+=head2 toggle_network
```

Comment:
I wonder if we can find a better name. "toggle" sounds like it would switch back and forth, not necessarily knowing if it was on or off in before. Maybe "switch_network"? Or separate "enable_network", "disable_network"?

Reply:
Had the same thought before submitting the PR: `switch_network` fits better here